src/client/components/MessageCenter/Topic/Topic.tsx

```diff
@@ -36,7 +36,7 @@ const Topic: React.FC<TopicProps> = (props) => {
   const user = useUser()
 
   let sectionName = useParams<{ sectionName: string }>()?.sectionName
-  if (topic.type === MessageTopicType.messageBoard) sectionName = 'messageBoard'
+  if (topic.type !== MessageTopicType.review) sectionName = topic.type
```

Comment:
Why do we set sectionName here ? Isn't it added to the topic instance ?

Reply:
no, it is not. the topic instance doesn't contain the sectionName.
sectionName is used only to save the activity in the log. When it is not specified it is based on MessageTopicType.